# Robot name goes here
robot_car_rev08:
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controller 1
  front_left_steering_controller:
    type: effort_controllers/JointPositionController 
    joint: front_left_steering_joint
    pid : {p: 250.0, i: 3.00, d: 1.5}

  # Controller 2
  front_right_steering_controller:
    type: effort_controllers/JointPositionController 
    joint: front_right_steering_joint
    pid : {p: 250.0, i: 3.00, d: 1.5}

  # Controller 3
  rear_wheel_controller:
    type: velocity_controllers/JointVelocityController 
    joint: rear_wheel_joint
    pid : {p: 1.5, i: 1.00, d: 0.0}


 